MANIFEST.in

```diff
@@ -2,7 +2,6 @@ recursive-include cpp *.cc *.h CMakeLists.txt
 recursive-include CMake *
 recursive-include extern *
 recursive-include freud *.py *.pyx *.pxd *.pxi CMakeLists.txt
-recursive-include requirements
```

Comment:
Why were these included before? If there was a good reason, why not package them now?